### ISSUE TITLE
Use collection variables failed

### ISSUE THREAD

**Issue by shiconsLeva** [a85/POSTMan-Chrome-Extension#10718]
### Is there an existing issue for this?

- [x] I have searched the tracker for existing similar issues

### Describe the Issue

![image](https://user-images.githubusercontent.com/52996325/156483189-a97fa10e-3521-4433-9a7d-400578675181.png)
![image](https://user-images.githubusercontent.com/52996325/156483340-fb17e215-6d22-42b4-be24-e53955b9fadf.png)
I set a collection variable named 'origin' but it can't be used when I tap the 'Send and Download' button

### Steps To Reproduce

I set a collection variable named 'origin' but it can't be used when I tap the 'Send and Download' button

### Environment Information

```markdown
- Operating System: Windows7
- Platform Type: Native App
- Postman Version: 9.14.8
```

**Comment by giridharvc7** [comment]
We are looking into this issue, will let you know once fixed is rolled out

**Comment by SchmidtWebmedia** [comment]
Some further information, I think this could be related to this issue here:

- The values from the "collection variables" do not overwrite the values from the "environment variables".
- The "Code Snippet" tool also does not take the values from the "collection variables".

**System**:
- Windows 10
- PostMan v9.14.9

**Comment by giridharvc7** [comment]
@SchmidtWebmedia We've raised a fix for this, and this should be solved now in v9.15.2+ , can you please check and let us know if it works for you? 
Feel free to reopen this ticket if not.
